jupyter-docker/Dockerfile

```diff
@@ -129,6 +129,7 @@ RUN apt-get install -yq --no-install-recommends \
  && pip install google-cloud \
  && pip install --ignore-installed firecloud==0.16.7 \
  && pip install -U scikit-learn \
+ && pip install statsmodels==0.8.0 \
```

Comment:
Weird that this is showing up in the diff, since it's already in dev. Do you need to rebase this branch?